proto/src/determined/trial/v1/trial.proto

```diff
@@ -204,7 +204,7 @@ message TrialMetrics {
   // The trial run associated with these metrics.
   int32 trial_run_id = 2;
   // The number of batches trained on when these metrics were reported.
-  int32 steps_completed = 3;
+  int32 steps_completed = 3;  // TODO: rename to batches. batches_trained?
```

Comment:
@hamidzr I think you had a few threads on slack about this rename. were there any decisions?